### PR TITLE
skill column change

### PR DESCRIPTION
**Pull request by Clarkbaum**
issue: https://github.com/sogetiprofileteam/profile/issues/46

changed skills from one long column to be 2 columns of 5

### CODE REVIEW

**Comment by NatThrock** [comment]
I like the simplicity. 

Because we read left to right, is there a way we could implement the for loop to generate columns like this?

| Col 1   | Col 2     |
--------------------
| Item 1 | Item 2   |
| Item 3 | Item 4   |
| Item 5 | Item 6   |
| Item 7 | Item 8   |
| Item 9 | Item 10 |

This may give some ideas: https://stackoverflow.com/questions/41597438/how-to-display-two-table-columns-per-row-in-angular-2-loop-angular2-equivalent

File: src/app/feature/consultant/components/consultant-technical-skills/consultant-technical-skills.component.html [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+icon="edit" 
+(actionClicked)="openEditSkillsDialog()">
 </app-consultant-section-header>
 
-<section *ngIf="technicalSkills$  | async as technicalSkills">
-  <ng-container *ngFor="let skill of technicalSkills">
-    <p>{{skill.name}}</p>
-  </ng-container>
+<section *ngIf="technicalSkills$ | async as technicalSkills">
+	<div class="container">
```
Comment (NatThrock) [inline src/app/feature/consultant/components/consultant-technical-skills/consultant-technical-skills.component.html]: The container div isn't necessary.

File: src/app/feature/consultant/components/consultant-core-skills/consultant-core-skills.component.html [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+title="Core Skills" 
+icon="edit" 
+(actionClicked)="openEditSkillsDialog()">
 </app-consultant-section-header>
 
 <section *ngIf="coreSkills$ | async as coreSkills">
-  <ng-container *ngFor="let skill of coreSkills">
-    <p>{{skill.name}}</p>
-  </ng-container>
+	<div class="container">
```
Comment (NatThrock) [inline src/app/feature/consultant/components/consultant-core-skills/consultant-core-skills.component.html]: The container div isn't necessary.

**Comment by Clarkbaum** [comment]
@NatThrock container class moved to parent element

**Pull request merged**